docs/reference/deprecated_api.rst

```diff
@@ -7,14 +9,39 @@ Deprecated Endpoints
 .. warning:: 
    The endpoints listed in this document are deprecated, and they will not be available in future versions of the Flocker API.
 
+The Flocker Container API
+=========================
+
 The Flocker Container API enables you to manage containers in your cluster.
 However, ClusterHQ is no longer developing any future features in this area, as you can now use Flocker with 3rd party orchestration tools.
 
-The deprecated endpoints listed below are still available for use, but please be aware that they will not be available in future versions of the Flocker API:
+.. note::
+   Flocker's container management features depend on Docker.
+   You will need to make sure `Docker (at least 1.8) is installed`_ and running before you enable ``flocker-container-agent``.
+
+.. _Docker (at least 1.8) is installed: https://docs.docker.com/installation/
+
+Enabling the Container Agent Service
+------------------------------------
+
+* On CentOS 7 and RHEL 7.2, run the following commands:
+  
+  .. prompt:: bash [root@agent-node]#
+  
+     systemctl enable flocker-container-agent
+     systemctl start flocker-container-agent
+
+* On Ubuntu, run the following command:
+
+  .. prompt:: bash [root@agent-node]#
+  
+     service flocker-container-agent start
 
 Deprecated Endpoints
 ====================
 
+The deprecated endpoints listed below are still available for use, but please be aware that they will not be available in future versions of the Flocker API:
+
```

Comment:
And move this (or something like) into the release notes file and advise users to refer to the 1.11.0 documentation for instructions on enabling the container agent and for container related API docs.